www/index.html

```diff
@@ -94,7 +94,7 @@ <h1>How to enable Dark Mode on Facebook, Amazon, YouTube, Google Search, Wikiped
     </main>
     <footer class="page-grid-footer">
         <nav>
-            <a href="https://github.com/darkreader/darkreader">Github</a>
+            <a href="https://github.com/darkreader/">GitHub</a>
```

Comment:
Thanks for finding this typo. But let's leave the link as it was.

Reply:
Fixed. 🙂